src/components/Filters/components/Date/helper.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withNamespaces } from 'react-i18next';
+import moment from 'moment';
+import { noop } from '../../../../helpers/utils';
+import { Accordion, Button, Header, Menu, Segment } from 'semantic-ui-react';
+
+import 'react-day-picker/lib/style.css';
+import { today } from '../../../../helpers/date';
+import FastDayPicker from './FastDayPicker';
+
+export const TODAY        = 'TODAY';
+export const YESTERDAY    = 'YESTERDAY';
+export const LAST_7_DAYS  = 'LAST_7_DAYS';
+export const LAST_30_DAYS = 'LAST_30_DAYS';
+export const CUSTOM_RANGE = 'CUSTOM_RANGE';
+export const CUSTOM_DAY   = 'CUSTOM_DAY';
+
+export const datePresets = [
+  TODAY,
+  LAST_7_DAYS,
+  LAST_30_DAYS
+];
+
+export const presetToRange = {
+  [TODAY]: () => {
+    const nToday = today().toDate();
+    return ({ from: nToday, to: nToday });
+  },
+  [LAST_7_DAYS]: () => ({
+    from: today().subtract(6, 'days').toDate(),
```

Comment:
Why subtract 6 for 7 days?

Reply:
- its copy/paste from usual kmedia date filter
- additional day is today